README.md

```diff
@@ -51,6 +51,11 @@ For more examples [check](https://github.com/DomCR/ACadSharp/tree/master/src/ACa
 
 For more information visit the :construction: [wiki](https://github.com/DomCR/ACadSharp/wiki) :construction: .
 
+Building
+---
+
+This project uses https://github.com/DomCR/CSUtilities as a Shared Project. Make sure to add the files to the ACadSharp\src\CSUtilities local directory when building.
```

Comment:
Good idea to add the building instructions, could you add this line?

Before building run `git submodule update --init --recursive` in the repository to load the submodules.

I think it gives a more clear instruction of how to fix it.

Thanks!